### ISSUE TITLE
I've made a updated version for Python3

### ISSUE THREAD

**Issue by jradxl** [GilGalaad/duplicity-rclone#7]
Thanks for your original work.\
I've updated it for Python3 as now Duplicity is Python3\
See:\
https://github.com/jradxl/duplicity-rclone

**Comment by GilGalaad** [comment]
Hi  @jradxl , thank you for your work.
FreeBSD is still on Phyton2.7 version, so I still didn't need to touch my code.
By the way, I have been contacted by the duplicity mantainer (look at closed issues of this project), and he's planning to merge this backend into duplicity itself :)

**Comment by kwloafman** [comment]
Hi,

rclonebackend.py in duplicity 0.8.09 is released for both py2 and py3, but
without any docs at the moment.  It runs as documents by GilGilaad.

I need to change to a non-copy command so that all the renames are not
needed, plus add to the man page.

...Thanks,
...Ken

**Comment by jradxl** [comment]
Thanks for your replies..
What a coincidence. If Duplicity had been in Github I would have used 8.09/latest, but looking at code.launchpad I was faced with a new learning curve I thought to use 8.08 which I'd already downloaded. At that point, I'd no idea whether I'd be able to get a development instance going!
Of course I tried the snap version first, but that is read-only, and Ubuntu's is old.
Serves me right for not going for latest dev version!
My beginner's code version is interestingly different from Ken's

Ken,
Is there any detailed documentation on the backend API. I searched around and didn't find any.

I might see if I can get Idrvebackend working, as I have one of their accounts!
Cheers
John

PS
Pycharm is really great!
